src/systemtest/java/com/ericsson/ei/systemtest/testexecutionflow/TestExecutionFlowSteps.java

```diff
@@ -0,0 +1,105 @@
+    public void configurations_are_provided() {
+        config.initEIFrontend();
+        config.initEIBackend();
+        config.initJenkinsConfig();
+        config.initRemRemConfig();
+    }
+
+    @Given("^a jenkins job '\\\"([^\\\"]*)\\\"' from '\"([^\"]*)\"' is created$")
+    public void a_jenkins_job_from_is_created(String jenkinsJobName, String scriptFileName) throws Throwable {
+        boolean success = StepsUtils.createJenkinsJob(
+                jenkinsJobName,
+                scriptFileName,
+                config.getJenkinsBaseUrl(),
+                config.getJenkinsUsername(),
+                config.getJenkinsPassword(),
+                JENKINS_TOKEN,
+                JENKINS_JOB_XML,
+                config.getRemremBaseUrl()
+         );
+
+        if (success) {
+            jenkinsJobNames.add(jenkinsJobName);
+        }
+
+        assertTrue("Failed to create jenkins job.", success);
+    }
+
+    @Then("^we continue with the next step$")
+    public void we_continue_with_the_next_step() {
+        //Just for cucumber to make sense
+    }
+
+    @Then("^subscriptions and jenkins jobs should be removed$")
+    public void subscriptions_and_jenkins_jobs_should_be_removed() throws Throwable {
+        StepsUtils.deleteJenkinsJobs(jenkinsJobNames);
+        StepsUtils.deleteSubscriptions(config.getEiFrontendBaseUrl(), config.getEiBackendBaseUrl());
+    }
+
+    @Given("^subscription object \"([^\"]*)\" is created which will trigger \"([^\"]*)\"$")
+    public void subscription_is_created(String subscriptionName, String nameOfTriggeredJob) throws Throwable {
+        StepsUtils.createSubscription(subscriptionName, nameOfTriggeredJob, config.getJenkinsUsername(), config.getJenkinsPassword(), config.getJenkinsBaseUrl());
+    }
+
+    @When("^notification with key \"([^\"]*)\" and value \"([^\"]*)\" is added to \"([^\"]*)\"$")
+    public void notification_with_key_and_value_is_added_to(String key, String value, String subscriptionName) throws Throwable {
+        StepsUtils.addNotificationToSubscription(key, value, subscriptionName);
+    }
+
+    @When("^condition with jmespath \"([^\"]*)\" is added to \"([^\"]*)\"$")
+    public void condition_with_jmespath_is_added_to(String jmesPath, String subscriptionName) throws Throwable {
+        StepsUtils.addConditionToRequirement(jmesPath, subscriptionName);
+    }
+
+    @Then("^we send the \"([^\"]*)\" to eiffel intelligence for creation\\.$")
+    public void we_send_the_to_eiffel_intelligence_for_creation(String subscriptionName) throws Throwable {
+        ResponseEntity response = StepsUtils.sendSubscriptionToEiffelIntelligence(subscriptionName, config.getEiFrontendBaseUrl(), config.getEiBackendBaseUrl());
+
+        assertEquals("Failed to create subscription. Response: " + response.getBody(), 200, response.getStatusCode());
+    }
+
+    @Given("^all previous tests passes\\.$")
+    public void all_previous_tests_passes() {
+        //Just for cucumber to make sense
```

Comment:
Why have all these empyt test step. Remove it and we will have shorter story to read in cucumber test step file.

Reply:
Same comment on all other similar places.

Reply:
Otherwise the Cucumber story which is meant to be read like a text wont make sense. It will be, "Given this, When that...."  instead of "Given this, When that, Then continue with next step" just to make sense when reading the test. I think it's worth it even if we need an empty generated cucumber test.

Reply:
I don't understand to have a test step that test nothing and always is marked successful. This will give a missleading test result from a statistically view, if someone is asking how many test we have and so on. Write a print out or something instead of a test step, if you think it make it easier to be read.

Reply:
We are talking about two different things. Yes, the test is redundant. What makes it useful if the way you read the test in cucumber. I read this on a [blog](https://medium.com/@kamil/why-i-write-cucumber-steps-with-step-definitions-that-have-no-code-824f70899eab) "This is why I think that instead of not writing steps with empty step definitions, you should make a habit of not writing steps that don’t bring any value to the users or the delivery team. These are two different things." and i agree.

In the test results, it will just say " Scenario " success. Each scenario is one test. Whats inside each scenario is just steps and isn't shown with JUnit.